### PR TITLE
[v3-1952 test 6] Disqualification is a one way ticket. Storage Node can't get around it.

### PR DESCRIPTION
**Pull request by Fadila82**
What: Take the disqualified storage node and give it a high uptime and audit rate. 

Why: To check that disqualification is permanent

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: pkg/audit/disqualification_test.go [hunk 1]
```diff
@@ -196,6 +196,45 @@ func TestDisqualifiedNodesGetNoUpload(t *testing.T) {
...
+		_, err = satellite.DB.OverlayCache().UpdateUptime(ctx, disqualifiedNode.ID(), true, 1, 1, 1)
+		require.NoError(t, err)
+
+		assert.True(t, isDisqualified(t, ctx, satellite, disqualifiedNode.ID()))
+
+		_, err = satellite.DB.OverlayCache().UpdateStats(ctx, &overlay.UpdateRequest{
+			NodeID:       disqualifiedNode.ID(),
+			IsUp:         true,
+			AuditSuccess: true,
+			AuditLambda:  1,
```
Comment (mobyvb) [inline pkg/audit/disqualification_test.go]: set audit lambda to 0 so that we forget about history

File: pkg/audit/disqualification_test.go [hunk 2]
```diff
@@ -196,6 +196,45 @@ func TestDisqualifiedNodesGetNoUpload(t *testing.T) {
...
+			AuditWeight:  1,
+			AuditDQ:      1,
+			UptimeLambda: 1,
```
Comment (mobyvb) [inline pkg/audit/disqualification_test.go]: same with uptime lambda. Set to 0. Maybe add a comment next to both cases explaining why (we want to erase previous history to make it "good")

File: pkg/audit/disqualification_test.go [hunk 3]
```diff
@@ -196,6 +196,45 @@ func TestDisqualifiedNodesGetNoUpload(t *testing.T) {
...
+
+		assert.True(t, isDisqualified(t, ctx, satellite, disqualifiedNode.ID()))
+
+		_, err = satellite.DB.OverlayCache().UpdateStats(ctx, &overlay.UpdateRequest{
+			NodeID:       disqualifiedNode.ID(),
+			IsUp:         true,
+			AuditSuccess: true,
+			AuditLambda:  1,
+			AuditWeight:  1,
+			AuditDQ:      1,
```
Comment (mobyvb) [inline pkg/audit/disqualification_test.go]: Set AuditDQ/UptimeDQ to 0 to guarantee that the new reputation scores are larger than the DQ thresholds. That way, the test is verifying that even though reputation is undoubtedly larger than the dq threshold, the node remains disqualified

File: pkg/audit/disqualification_test.go [hunk 4]
```diff
@@ -196,6 +196,45 @@ func TestDisqualifiedNodesGetNoUpload(t *testing.T) {
 	})
 }
 
+func TestDisqualifiedNodesRemainsDisqualified(t *testing.T) {
```
Comment (kaloyan-raev) [inline pkg/audit/disqualification_test.go]: typo: "DisqualifiedNode" instead of "DisqualifiedNodes".

**Pull request merged**